README.md

```diff
@@ -22,14 +19,10 @@
 **Basic Installation**
```

Comment:
Maybe we should change this to: "How to run" or something similar?

Reply:
Sounds good to me.

Reply:
@nabaco Done.